components/viz/service/gl/gpu_service_impl.cc

```diff
@@ -856,10 +856,14 @@ void GpuServiceImpl::CreateGpuMemoryBuffer(
     int client_id,
     gpu::SurfaceHandle surface_handle,
     CreateGpuMemoryBufferCallback callback) {
+#if BUILDFLAG(ENABLE_COBALT_HERMETIC_HACKS)
```

Comment:
Please add a comment here to make it clear that this is returning a stub, not a valid default value. Also please add TODOs with bugs for both of these changes